.github/workflows/tests.yml

```diff
@@ -70,3 +70,53 @@ jobs:
         with:
           name: test_results_${{ matrix.api-versions }}
           path: '**/build/test-results/**/TEST-*.xml'
+
+  instrumetation-tests:
+    runs-on: macos-latest
```

Comment:
I want to test behaviors on as much as possible Android devices. Some methods has restrict usage for different Android versions, so more Android version checking can give more fidelity for compatibility. It actually increases CI run time, and delay other CI jobs because of free machine instance limits. What about reducing two SDK levels? The latest one, and one with 2-3 years different.

Reply:
If there is only one checking in sequence, it doesn't reduce any time, because they are parallel under GitHub's limition. But if there are many checking in sequence, for example, many rebasing for PRs, it will reduce about almost half waiting time for macos instances at average, in my opinion.

Reply:
Reduced to 28 and 30.